electroncash/rpa/paycode.py

```diff
@@ -243,15 +243,16 @@ def generate_transaction_from_paycode(wallet, config, amount, rpa_paycode=None,
         # Generate the raw transaction
         raw_tx_string = tx.as_dict()["hex"]
 
-        # Get the TxId for this raw Tx.
-        double_hash_tx = bytearray(sha256(sha256(bytes.fromhex(raw_tx_string))))
-        double_hash_tx.reverse()
-        txid = double_hash_tx.hex()
+        # Get the 'txinid' (double hash of serialized input) for this raw Tx.
+        txin= tx._inputs[0]
```

Comment:
Space around operator. (did you install PyCharm?)

Reply:
fixed, yes.